jetty-osgi/jetty-osgi-boot/jettyhome/etc/jetty.xml

```diff
@@ -95,7 +95,7 @@
          <Item>org.eclipse.jetty.plus.webapp.EnvConfiguration</Item>
          <Item>org.eclipse.jetty.webapp.JmxConfiguration</Item>
          <Item>org.eclipse.jetty.websocket.server.JettyWebSocketConfiguration</Item>
-         <Item>org.eclipse.jetty.websocket.jsr356.JavaxWebSocketExtensionConfig</Item>
+         <Item>JavaxWebSocketExtensionConfig</Item>
```

Comment:
This is wrong.

`JavaxWebSocketExtensionConfig` is not a `Configuration` class.